debs/SPECS/wazuh-manager/debian/postinst

```diff
@@ -71,6 +72,13 @@ case "$1" in
         chown ${USER}:${GROUP} ${DIR}/queue/db/global.db*
     fi
 
+    if [ -f "${DIR}/${VD_FILENAME}" ]; then
+        tar -xf ${DIR}/${VD_FILENAME} -C ${DIR}
+        chown ${USER}:${GROUP} ${DIR}/queue/vd
+        chown ${USER}:${GROUP} ${DIR}/queue/vd-updater
```

Comment:
This folder isn't required.

Reply:
According to the requirements, the vd_updater folder must be compressed.

Reply:
Ok, thank you.